.build-test-rules.yml

```diff
@@ -42,3 +42,9 @@ catch2/examples/catch2-console:
   disable:
     - if: IDF_VERSION_MAJOR < 5
       reason: Example relies on WHOLE_ARCHIVE component property which was introduced in IDF v5.0
+
+led_strip/examples/lcd_qemu_rgb_panel:
+  enable:
+    - if: (IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR >= 3) or (IDF_VERSION_MAJOR > 5)
+    - if: IDF_TARGET in ["esp32", "esp32c3"]
```

Comment:
I think you need to combine two clauses into on `if`, otherwise it will be built _either_ for IDF >= 5.3 _or_ esp32 || esp32c3.

Reply:
I thought it was interpreted as an `and` :smile: 
I am going to correct it

Reply:
Corrected